src/utils/logSort.js

```diff
@@ -0,0 +1,3 @@
+export default function logSort(entryA, entryB) {
+  return Date.parse(entryA.created) < Date.parse(entryB.created);
```

Comment:
When I was testing this I was getting some events out of order so we're sorting each time here. Something more clever should probably be done. e.g. 
```py
entries = [{created: 2:30}, {created: 4:15}]
receive new logEntry a {created: 3:00}, 
endEntry = []
iterate backwards through entries, get entry b
  endEntry.unshift(b)
  if a.created < b.created: endEntry.unshift(a), break
# endEntry = [a, {created: 4:15}]
entries.splice(0, entries.length - endEntry.length) # entries  = [{created: 2:30}]
entries = entries.concat(endEntry) #entrie = [{created: 2:30}, a, {created: 4:15}]
```
Which is probably faster than sorting them every time especially as the logs get longer?